render/src/lib/services/service.ts

```diff
@@ -45,6 +59,8 @@ export class Service extends Toolkit.APluginService {
     private _chartLimit = 30;
     private _limit = 300;
     private _sessions: string[] = [];
+    private _configSettings: {[path: string]: IOptions};
```

Comment:
Probably like it?
```
private _configSettings: {[path: string]: IOptions} | undefined;
```
because in constructor there are a few possibilities to have it as undefined